packages/react-native/src/Authenticator/Defaults/SignIn/SignIn.tsx

```diff
@@ -1,13 +1,95 @@
 import React from 'react';
-import { Text } from 'react-native';
+import { StyleSheet, TextStyle, View, ViewStyle } from 'react-native';
+
+import {
+  DefaultFooter,
+  DefaultFormFields,
+  DefaultHeader,
+  FederatedProviderButtons,
+} from '../../common';
+
+import { Button, ErrorMessage, Tab, Tabs } from '../../../primitives';
 
-import { DefaultFooter } from '../../common/DefaultFooter';
-import { DefaultHeader } from '../../common/DefaultHeader';
-import { DefaultFormFields } from '../FormFields';
 import { DefaultSignInComponent } from '../types';
 
-const SignIn: DefaultSignInComponent = () => {
-  return <Text>SignIn</Text>;
+// TODO: clean these up once primitive theming is in place
+interface DefaultSignInStyle {
+  buttonPrimary: ViewStyle;
+  buttonPrimaryLabel: TextStyle;
+  buttonSecondary: ViewStyle;
+  buttonSecondaryLabel: TextStyle;
+  container: ViewStyle;
+  tabs: ViewStyle;
+}
+
+const styles: DefaultSignInStyle = StyleSheet.create({
+  container: {
+    justifyContent: 'center',
+    flex: 1,
+    padding: 16,
+  },
+  buttonPrimary: {
+    backgroundColor: 'teal',
+    marginVertical: 8,
+    paddingVertical: 12,
+  },
+  buttonPrimaryLabel: { color: 'white', fontSize: 16, fontWeight: 'bold' },
+  buttonSecondary: {
+    marginVertical: 8,
+    paddingVertical: 12,
+  },
+  buttonSecondaryLabel: { color: 'teal' },
+  tabs: { marginBottom: 8 },
+});
+
+const SignIn: DefaultSignInComponent = ({
+  error,
+  fields,
+  Footer,
+  FormFields,
+  Header,
+  hideSignUp,
+  isPending,
+  socialProviders,
+  toResetPassword,
+  toFederatedSignIn,
+  toSignUp,
+}) => {
+  return (
+    <View style={styles.container}>
+      {hideSignUp ? null : (
+        <Tabs style={styles.tabs}>
+          <Tab>Sign In</Tab>
```

Comment:
Now that Will's shared Authenticator text PR is in `rna/main`, can we use it for strings?